src/Parser/DeathKnight/Blood/Modules/Talents/Ossuary.js

```diff
@@ -19,6 +23,17 @@ class Ossuary extends Analyzer {
     return this.combatants.getBuffUptime(SPELLS.OSSUARY.id) / this.owner.fightDuration;
   }
 
+  on_byPlayer_cast(event) {
+
+    if (event.ability.guid === SPELLS.DEATH_STRIKE.id) {
```

Comment:
It's preferred that you avoid nested if statements when possible. You can instead check for the opposite and return early.